plugins/org.jkiss.dbeaver.ext.sqlite/src/org/jkiss/dbeaver/ext/sqlite/model/SQLiteTable.java

```diff
@@ -156,8 +157,8 @@ private boolean obtainIsWithoutRowId(@NotNull DBRProgressMonitor monitor) throws
         }
     }
 
-    private boolean hasPrimaryKey() throws DBException {
-        List<GenericUniqueKey> constraints = getConstraints(new VoidProgressMonitor());
+    public static boolean hasPrimaryKey(DBSTable table) throws DBException {
```

Comment:
Also an over-complicated check. Do we really need it?
Can we just return pseudo attrs like they are always there?